installers-conda/resources/post-install.sh

```diff
@@ -106,37 +118,54 @@ if [[ -n "$shell_init" ]]; then
 fi
 
 # ----
-if [[ $OSTYPE = "linux"* ]]; then
+if [[ "$OSTYPE" = "linux"* ]]; then
     cat <<EOF
 
+###############################################################################
+#                             !!! IMPORTANT !!!
 ###############################################################################
 Spyder can be launched by standard methods in Gnome and KDE desktop
-environments. Additionally, Spyder can be launched in Gtk-based desktop
-environments (e.g. Xfce) from the command line:
-
-$ gtk-launch spyder
-
-Spyder can also be launched from the command line for all Linux variants by:
+environments. It can also be launched from the command line on all Linux
+distros with the command:
 
 $ spyder
 
 To uninstall Spyder, run the following from the command line:
 
 $ uninstall-spyder
 
-#####################
-# !!! IMPORTANT !!! #
-#####################
-
-The spyder and uninstall-spyder commands will only be available in new shell
-sessions. To make them available in this session you must source your .bashrc
-file with:
+These commands will only be available in new shell sessions. To make them
+available in this session, you must source your $shell_init file with:
 
-$ source ~/.bashrc
+$ source $shell_init
 
 ###############################################################################
 
 EOF
 fi
 
 echo "*** Post install script for ${INSTALLER_NAME} complete"
+
+# ----
+[[ -n "$CI" ]] && exit 0  # Running in CI, don't launch Spyder
+
+if [[ "$OSTYPE" = "darwin"* ]]; then
+    tmp_dir=${SHARED_INSTALLER_TEMP}/spyder
+    launch_script=${tmp_dir}/post-install-launch.sh
+    echo "Creating post-install launch script ..."
+    mkdir -p $tmp_dir
+    cat <<EOF > $launch_script
+#!/bin/bash
+while pgrep -fq Installer.app; do
+    sleep 1
+done
+open -a $shortcut_path
+EOF
+    chmod +x $launch_script
+
+    nohup $launch_script &>/dev/null &
+elif [[ -n "$(which gtk-launch)" ]]; then
+    gtk-launch $(basename $shortcut_path)
+else
+    $spy_exe
```

Comment:
```suggestion
    nohup $spy_exe $>/dev/null &
```
I think we may need this so that Spyder is not killed when the installer exits, if not launched with gtk-launch.

Reply:
Let me check and I'll get back to you.

Reply:
I should clarify: when the shell exits, not just the installer.

The following only applies if `gtk-launch` is _not_ used. If the user launches the installer from the commandline, after the installer is done and Spyder is launched, the shell session remains open; when the user closes this shell session, Spyder will be killed. If the user runs the installer from the file explorer (right-click -> "Run as program"), then a new shell session opens, runs the installer, launches Spyder, and exits, killing Spyder before the user even knows it opens.

Reply:
In my case, after running the installer in a shell, closing it didn't close Spyder but blocked the shell. So, I think using `nohup` is necessary in this case.